docs/README.md

```diff
@@ -214,6 +214,29 @@ This file can be moved from system to system, just like OpenPGP or SSH key sets.
 - To create a signing key for a client, use `bindle create-key`
 - By default, if Bindle does not find an existing keyring, it creates one of these when it first starts.
 
+## Generating Change Events
+
+Starting bindle with the `--events`  flag will cause it to emit an record of the event to a file named `bindle-event.log` in the bindle data directory.
```

Comment:
We might want to have a note about using some sort of tool like `logrotate` (or we should provide one) to avoid getting an infinitely growing file